src/utilities/reactionbookmark.go.tmpl

```diff
@@ -0,0 +1,39 @@
+{{/*
+	Allows you to bookmark messages via a reaction.
+	See <https://yagpdb-cc.github.io/utilities/reactionbookmark> for more information.
+
+        Author: TheHolyWatermelon <https://github.com/TheHolyWaterme1on>
+	Co-Author: Luca Zeuch <https://github.com/l-zeuch>
+
+*/}}
+
+{{/* Configuration start */}}
+{{ $EMOJI := "🔖" }} {{/* The emoji used to trigger a bookmark. use the emoji name when using a custom emoji. */}}
+{{/* Configuration end */}}
+
+{{/* Actual Code */}}
+{{ if and (eq .Reaction.Emoji.Name $EMOJI) .ReactionAdded }}
```

Comment:
Indent the block associated with this `if`.

Reply:
Done!